engine/cdn/storage/storageunit.go

```diff
@@ -378,7 +378,7 @@ func (r *RunningStorageUnits) Start(ctx context.Context, gorts *sdk.GoRoutines)
 	// 	Feed the sync processes with a ticker
 	gorts.Run(ctx, "RunningStorageUnits.Start", func(ctx context.Context) {
 		tickr := time.NewTicker(time.Duration(r.config.SyncSeconds) * time.Second)
-		tickrPurge := time.NewTicker(30 * time.Second)
+		tickrPurge := time.NewTicker(5 * time.Second)
```

Comment:
Can you move this in config with a default value ?

Reply:
done